### ISSUE TITLE
[Request] Avoid repeated songs in history

### ISSUE THREAD

**Issue by srvanrell** [mps-youtube/mps-youtube#504]
When I was using the new `history` command I found that songs may be repeated several times in the list. I would prefer to just have record of the last time that I played a song. What do you think?

If you agree, I can look into it and gladly make a PR. Let me know.

**Comment by kraetzin** [comment]
I feel as though the expected function of history is all of the played songs in order, although I may be slightly biased. ;) I'd at least like to retain that functionality and maybe implement the new one as an extended history command, ie. something like `history filtered`, or move the old functionality to an extended command.

**Comment by ids1024** [comment]
> I feel as though the expected function of history is all of the played songs in order

 I agree with this. The functionality requested here is more "recent" songs than "history", though I am not sure it would be a good idea to add a seperate command with a completely different name for a slight variation on functionality.

**Comment by srvanrell** [comment]
@ids1024 you are right! "recent" fit to the functionality that I described above. 
Besides of the name, do you agree with "recent" functionallity addition?

**Comment by ids1024** [comment]
Yes, the functionality should probably be available in some way.

**Comment by srvanrell** [comment]
I added a proposal of recent functionality in PR #510 
Let me know about any change that need to be done

**Comment by SanketDG** [comment]
This doesnt work at all as proposed. @srvanrell 

Each of the videos in history is assigned a different id, so no videos in history are same.

**Comment by SanketDG** [comment]
This diff works, where we check by the youtube unique id.
```
-            hist_list = [x for x in hist_list if not (x in seen or seen_add(x))]
+            hist_list = [x for x in hist_list if not (x.ytid in seen or seen_add(x.ytid))]
```

**Comment by srvanrell** [comment]
I'm sure that it worked at that time. Perhaps, something has changed in the from that time.

**Comment by SanketDG** [comment]
Interesting, maybe the `Video` abstraction changed, nonetheless I have opened https://github.com/mps-youtube/mps-youtube/pull/772
